<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-01-17T19:45:23 IST" name="Default suite" finished-at="2025-01-17T19:45:58 IST" duration-ms="34805">
    <groups>
    </groups>
    <test started-at="2025-01-17T19:45:23 IST" name="Default test" finished-at="2025-01-17T19:45:58 IST" duration-ms="34805">
      <class name="testSuits.ChangePasswordTestSuite">
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:testSuits.ChangePasswordTestSuite@4145bad8]" started-at="2025-01-17T19:45:24 IST" name="launchBrowser" finished-at="2025-01-17T19:45:37 IST" duration-ms="13076" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="login()[pri:0, instance:testSuits.ChangePasswordTestSuite@4145bad8]" started-at="2025-01-17T19:45:37 IST" name="login" finished-at="2025-01-17T19:45:39 IST" duration-ms="2847" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="TC123_ChangePasswordTest()[pri:0, instance:testSuits.ChangePasswordTestSuite@4145bad8]" started-at="2025-01-17T19:45:39 IST" name="TC123_ChangePasswordTest" finished-at="2025-01-17T19:45:55 IST" duration-ms="15406" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[TC123_ChangePasswordTest - Test Case failed. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: TC123_ChangePasswordTest - Test Case failed. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at testSuits.ChangePasswordTestSuite.TC123_ChangePasswordTest(ChangePasswordTestSuite.java:19)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC123_ChangePasswordTest -->
        <test-method is-config="true" signature="logout()[pri:0, instance:testSuits.ChangePasswordTestSuite@4145bad8]" started-at="2025-01-17T19:45:55 IST" name="logout" finished-at="2025-01-17T19:45:56 IST" duration-ms="1340" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:testSuits.ChangePasswordTestSuite@4145bad8]" started-at="2025-01-17T19:45:56 IST" name="closeBrowser" finished-at="2025-01-17T19:45:58 IST" duration-ms="2000" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- testSuits.ChangePasswordTestSuite -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
